source/plugins/go/src/oms.go

```diff
@@ -475,6 +489,40 @@ func updateContainerImageNameMaps() {
 	}
 }
 
+func updateContainerLogV2ExtensionMaps(isWindows bool) {
+	for ; true; <-ContainerLogV2ExtensionConfigRefreshTicker.C {
+		Log("updateContainerLogV2ExtensionMaps::Info: Invoking GetInstance for GetContainerLogV2ExtensionConfig")
+		maxRetries := 3
+		for attempt := 1; attempt <= maxRetries; attempt++ {
+			_namespaceStreamIdsMap, _streamIdNamedPipeMap, err := extension.GetInstance(FLBLogger, ContainerType).GetContainerLogV2ExtensionConfig(IsWindows)
+			if err != nil {
+				Log("updateContainerLogV2ExtensionMaps::error: %s attempt: %d", string(err.Error()), attempt)
+				time.Sleep(time.Duration(attempt+1) * time.Second)
+			} else {
+				Log("updateContainerLogV2ExtensionMaps:Info:Locking to update NamespaceStreamIdsMap and StreamIdNamedPipeMap")
+				ContainerLogV2ExtensionMapUpdateMutex.Lock()
+				for key := range NamespaceStreamIdsMap {
+					delete(NamespaceStreamIdsMap, key)
+				}
+				for key, value := range _namespaceStreamIdsMap {
+					NamespaceStreamIdsMap[key] = value
```

Comment:
can this be replaced with direct assign? NamespaceStreamIdsMap = _namespaceStreamIdsMap